scripts/gatkcondaenv.yml.template

```diff
@@ -43,8 +43,8 @@ dependencies:
   - scikit-learn==0.19.1
   - scipy==1.0.0
   - six==1.11.0
-  - $tensorFlowDependency
+  - tensorflow==1.9.0
```

Comment:
The `tensorFlowDependency` variable still exists in `build.gradle`, but due to this change its no longer honored. I think if you just completely revert all the changes in this file everything will be right again.